cypress/integration/AppAnalytics/DiseaseAnalytics.feature

```diff
@@ -4,21 +4,21 @@ Feature: Disease listing page analytics
         And browser waits
         Then there should be an analytics event with the following details
             | key                                         | value                                                      |
+					  | event                                       | TrialListingApp:Load:Results                               |
             | type                                        | PageLoad                                                   |
-            | event                                       | TrialListingApp:Load:Results                               |
-            | page.name                                   | www.cancer.gov/C4872                                       |
-            | page.title                                  | Breast Cancer Clinical Trials     							           |
-            | page.metaTitle                              | Breast Cancer Clinical Trials - National Cancer Institute  |
-            | page.language                               | english                                                    |
-            | page.type                                   | nciAppModulePage                                           |
-            | page.channel                                | Clinical Trials                                            |
+						| page.publishedDate                          | 02/02/2011                                                 |
             | page.contentGroup                           | Clinical Trials: Custom                                    |
-            | page.publishedDate                          | 02/02/2011                                                 |
-            | page.additionalDetails.numberResults        | (int)925                                                   |
-            | page.additionalDetails.trialListingPageType | disease					                                           |
+						| page.name                                   | www.cancer.gov/C4872                                       |
+					  | page.title                                  | Breast Cancer Clinical Trials     							           |
+						| page.channel                                | Clinical Trials                                            |
+					  | page.type                                   | nciAppModulePage                                           |
+					  | page.language                               | english                                                    |
+						| page.metaTitle                              | Breast Cancer Clinical Trials - National Cancer Institute  |
 						| page.additionalDetails.diseaseName			    | breast cancer		                                           |
+            | page.additionalDetails.trialListingPageType | disease					                                           |
+						| page.additionalDetails.numberResults        | (int)938                                                   |
```

Comment:
You need to add at least one integration test for a pager inside DiseasePrettyUrl.feature ( no need to have full functional test, just take the first scenario from AdultMetastaticBrainTumor.feature and implement it for breast-cancer path)

Reply:
Will do